src/serde.rs

```diff
@@ -0,0 +1,111 @@
+#[cfg(feature = "serialize")]
+use serde::{Serialize, Serializer};
+#[cfg(feature = "deserialize")]
+use serde::{Deserialize, Deserializer};
+use crate::{RefCnt, ArcSwapAny, Strategy};
+
+#[cfg(feature = "serialize")]
+/// # Examples
+///
+/// ```rust
+/// use arc_swap::ArcSwap;
+///
+/// #[derive(serde::Serialize)]
+/// struct Foo {
+///     field0: usize,
+///     field1: String,
+/// }
+///
+/// let data = Foo {
+///     field0: 123,
+///     field1: "123".to_owned(),
+/// };
+///
+/// let data_json = serde_json::to_string(&data).unwrap();
+/// println!("{}", data_json);
+/// ```
```

Comment:
Honestly, I don't think examples are necessary for common trait implementation. Besides, this example doesn't even use ArcSwap in any way as far as I can see. Let's just drop them (both, I'd suggest).

Reply:
done